Dockerfile

```diff
@@ -1,16 +1,29 @@
-FROM alpine:3.9.2
+FROM quay.io/openshift/origin-cli:v4.0.0
 
-ARG kubectlversion=1.12.4
-ARG yqversion=2.2.1
+ARG ocpythonlibver=0.8.6
 
-RUN apk add --no-cache curl bash
 RUN \
-  cd /usr/local/bin             && \
-  echo "Grabbing binaries..." && \
-  curl -sLO https://storage.googleapis.com/kubernetes-release/release/v${kubectlversion}/bin/linux/amd64/kubectl && \
-  curl -sLO https://github.com/mikefarah/yq/releases/download/${yqversion}/yq_linux_amd64 && \
-  mv /usr/local/bin/yq_linux_amd64 /usr/local/bin/yq && \
-  chmod +x /usr/local/bin/kubectl /usr/local/bin/yq
-ADD fetch.sh /usr/local/bin
+  cd /tmp && \
+  curl -L https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+  python /tmp/get-pip.py && \
+  pip install -U setuptools
+
+RUN \
+  cd /tmp && \ 
+  curl -LO https://github.com/openshift/openshift-restclient-python/archive/v${ocpythonlibver}.tar.gz && \
+  tar xvzf v${ocpythonlibver}.tar.gz && \
+  cd openshift-restclient-python-${ocpythonlibver} && \
+  (python setup.py install || python setup.py install)
```

Comment:
Should both commands here be identical?

Reply:
Yeah, there's some odd error that "fixes" by running the command again. I don't want to sink more time to fix any underlying issue so this hack will do it.

Reply:
ouch ok!

Reply:
The error, by the way is (from the first `python setup.py install`) is:

```
error: The 'requests' distribution was not found and is required by kubernetes
```

My unfamiliarity with Python leaves me with this hackaround.